src/common/styles/mixins.sass

```diff
@@ -38,24 +39,88 @@
   font-weight: 600 !important
   line-height: 150% !important
 
+@mixin h3-raleway
+  font-family: "Raleway", sans-serif
+  font-style: normal
```

Comment:
remove all font styles, because CSS already set this default to "normal"

Reply:
removed.